server_spidercast/src/MCC_RUM/src/rumFireOutThread.c

```diff
@@ -325,11 +325,12 @@ int send_single_packet(ConnInfoRec *cInfo, StreamInfoRec_T *pSinf, int rate_limi
   }
   else
   {
-    if ( isOdata && !pSinf->keepHistory)  
+    if ( isOdata && !pSinf->keepHistory)  {
       MM_put_buff(rmmTRec[inst]->DataBuffPool, packet) ;
       llmSimpleTraceInvoke(tcHandle,LLM_LOGLEV_ERROR,MSG_KEY(3272),"%llx%llx",
           "The packet is not valid. Additional information: wrInfo.bptr={0}, wrInfo.buffer={1}",
           LLU_VALUE(cInfo->wrInfo.bptr), LLU_VALUE(cInfo->wrInfo.buffer));
+    }
```

Comment:
again who knows what this is meant to do

Reply:
This was also done to resolve the misleading-indentation warning. I guess this change can be avoided.

Reply:
I think this is an unintended change of behaviour, instead the llmSimpleTraceInvoke() should be unindented (compare with lines 305-307